### PR TITLE
remove fs-extra dependency

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

File: node-tests/blueprints/route-test.js [hunk 1]
```diff
@@ -395,7 +391,6 @@ describe('Acceptance: ember generate and destroy route', function() {
     var args = ['route', 'application', '--pod'];
 
     return emberNew()
-      .then(() => remove(path.resolve('app/templates/application.hbs')))
```
Comment (rwjblue) [inline node-tests/blueprints/route-test.js]: @kellyselden - Do we just not need to remove this file?

File: package.json [hunk 2]
```diff
@@ -55,7 +55,6 @@
     "ember-cli-valid-component-name": "^1.0.0",
     "ember-cli-version-checker": "^1.3.1",
     "ember-router-generator": "^1.2.3",
-    "fs-extra": "^4.0.1",
```
Comment (rwjblue) [inline package.json]: Even if we kept it, I think it should become a devDep anyways

File: node-tests/blueprints/route-test.js (hunk 1 above)
Comment (kellyselden) [reply node-tests/blueprints/route-test.js]: Yea. I tried converting to `fs.unlink` first, but the file is never there. Must be legacy, and can come back in the future if needed.

File: package.json (hunk 2 above)
Comment (kellyselden) [reply package.json]: Agreed

**Comment by rwjblue** [comment]
Thanks @kellyselden !

**Pull request merged**